src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -15,6 +15,8 @@ internal static class SecurityProtocol
         public const SslProtocols DefaultSecurityProtocols =
             SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
 
+        public const SslProtocols SystemDefaultSecurityProtocols = SslProtocols.None;
```

Comment:
> if the protocols value comes in as SslProtocols.None, we should just let libcurl do whatever it wants to do

@stephentoub Yes, that's correct. I recommend ensuring that the default includes TLS 1.2.